fiware-platform/templates/_helpers.tpl

```diff
@@ -0,0 +1,4 @@
+{{/* vim: set filetype=mustache: */}}
```

Comment:
Is it necessary?

Reply:
shit, dont merge! Changes from another story were pulled in,  my bad..